### PR TITLE
Datn 70/telegram

### PR DESCRIPTION
**Pull request by btkhoi2001**
Update .env before merging to development @Nambe1110

### CODE REVIEW

File: src/components/channel/telegram/bot/telegramBotService.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default class TelegramUserService {
+  static async createConnection(token) {
+    const connection = "";
+
+    return connection;
+  }
+}
```
Comment (Loc-Tran-2111) [inline src/components/channel/telegram/bot/telegramBotService.js:7]: It seems redundant.

File: src/components/customer/customerController.js [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
-    const members = await customerService.getCompanyCustomers({
-      user: req.user,
-      limit,
-      page,
+  try {
+    const customers = await CustomerService.getCustomers({
+      companyId: user.company_id,
+      page: parseInt(page),
+      limit: parseInt(limit),
     });
```
Comment (Loc-Tran-2111) [inline src/components/customer/customerController.js:13]: If `parseInt` return `NaN`, will our app crash?

File: src/components/message/messageController.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  const { user } = req;
+  const { threadId } = req.params;
+  const { lastMessageId, limit = 20 } = req.query;
+
+  try {
+    const messages = await MessageService.getMessages({
+      companyId: user.company_id,
+      threadId: parseInt(threadId),
+      lastMessageId: parseInt(lastMessageId),
+      limit: parseInt(limit),
```
Comment (Loc-Tran-2111) [inline src/components/message/messageController.js:14]: Same here.

File: src/components/channel/telegram/bot/telegramBotService.js (hunk 1 above)
Comment (btkhoi2001) [reply src/components/channel/telegram/bot/telegramBotService.js:7]: Bot Telegram is not completed. I forgot to undo "git add" it.

File: src/components/customer/customerModel.js [hunk 4]
```diff
@@ -1,83 +1,65 @@
...
     address: {
       type: DataTypes.STRING,
-      allowNull: true,
     },
     note: {
       type: DataTypes.STRING,
-      allowNull: true,
     },
-    image_url: {
-      type: DataTypes.STRING,
```
Comment (Nambe1110) [inline src/components/customer/customerModel.js:80]: Hi Khoi,
Whether a company has image_url ?
image_url: {
      type: DataTypes.STRING,
      default:
        "https://uservice.cloud/api/image/uservice-default-company-avatar.png",
    },
BR,
Nam

File: src/components/customer/customerController.js (hunk 2 above)
Comment (Nambe1110) [reply src/components/customer/customerController.js:13]: We already have pagination for other api on development now. Thus, please take a look.

File: src/components/customer/customerApi.js [hunk 5]
```diff
@@ -9,10 +9,8 @@ customerRouter.use("/", (req, res, next) => {
   next();
 });
 
-customerRouter.get(
-  "/company-customers",
-  verifyToken.verifyToken,
-  getCompanyCustomers
```
Comment (Nambe1110) [inline src/components/customer/customerApi.js:15]: This api is requested by mobile and front-end teams. I'm not sure whether they use it or not. Thus, removing is quite risky

Comment (Loc-Tran-2111) [reply src/components/customer/customerApi.js:15]: Seem like it is just a rename. As I see, Khoi updated it on api docs.

**Pull request merged**